opslevel/resource_opslevel_property_definition.go

```diff
@@ -61,13 +74,37 @@ func resourcePropertyDefinitionRead(d *schema.ResourceData, client *opslevel.Cli
 	if err := d.Set("name", resource.Name); err != nil {
 		return err
 	}
+	if err := d.Set("description", resource.Description); err != nil {
+		return err
+	}
 	if err := d.Set("schema", resource.Schema.ToJSON()); err != nil {
 		return err
 	}
+	if err := d.Set("property_display_status", string(resource.PropertyDisplayStatus)); err != nil {
+		return err
+	}
 
 	return nil
 }
 
+func resourcePropertyDefinitionUpdate(d *schema.ResourceData, client *opslevel.Client) error {
+	id := d.Id()
+	input := opslevel.PropertyDefinitionInput{
+		Name:                  d.Get("name").(string),
+		Description:           d.Get("description").(string),
+		Schema:                opslevel.NewJSON(d.Get("schema").(string)),
+		PropertyDisplayStatus: opslevel.PropertyDisplayStatusEnum(d.Get("property_display_status").(string)),
+	}
+
+	_, err := client.UpdatePropertyDefinition(id, input)
```

Comment:
this is a thing???  I thought definitions only had create and delete?

Reply:
@rocktavious This was added very recently! We also have CLI supporting `update` on property definitions now. https://github.com/OpsLevel/cli/pull/225

Reply:
🎉